### PR TITLE
stoa0292b.stoa001_cts_opp-lat1/2

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated cts metadata
Updated edition metadata and headers to new standards.

### CODE REVIEW

File: data/stoa0292b/stoa001/stoa0292b.stoa001.opp-lat1.xml [hunk 1]
```diff
@@ -2,7 +2,9 @@
 <teiHeader>
     <fileDesc>
       <titleStmt>
-        <title>Corpus Scriptorum Ecclesiasticorum Latinorum (CSEL)</title>
+        <title>Alethia</title>
```
Comment (lcerrato) [inline data/stoa0292b/stoa001/stoa0292b.stoa001.opp-lat1.xml:5]: The title is marked as lat above.

**Review by lcerrato — CHANGES_REQUESTED** [review]

**Comment by lcerrato** [comment]
@AlisonBabeu 
Is this Alethia or Aletheia?

File: data/stoa0292b/stoa001/stoa0292b.stoa001.opp-lat1.xml (hunk 1 above)
Comment (AlisonBabeu) [reply data/stoa0292b/stoa001/stoa0292b.stoa001.opp-lat1.xml:5]: Hi @lcerrato I will fix the title and it is Alethia not Aletheia.

**Comment by lcerrato** [comment]
@AlisonBabeu Ok, I hadn't seen that transliteration for the Greek word. Looks like the wiki is wrong — I see all of the other sources have this spelling. Shame on me for looking there.

**Review by lcerrato — APPROVED** [review]

**Pull request merged**